nixos/roles/statshost/global-metrics.nix

```diff
@@ -12,32 +12,40 @@ let
   globalAllowedMetrics = [
```

Comment:
We probably also want `amd_rocm_smi` in here, for metrics generated by the GPU servers.